RoboticonColony/Assets/Code/Tile.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+sealed public class Tile
+{
+    public int Cost { get; private set; }
+    public int OwnerID { get; private set; }
+
+    private Dictionary<ItemType, int> _Resources;
+    private int _ownerID;
+
+    /// <summary>
+    /// Constructor for tile
+    /// </summary> 
+    /// <param name="cost">Cost of the tile</param>
+    /// <param name="ore">Ore that the tile has</param>
+    /// <param name="power">Power that the tile has</param>
+    public Tile(int cost, int ore, int power)
+    {
+        Cost = cost;
+        _Resources = new Dictionary<ItemType, int>();
+        _Resources[ItemType.Ore] = ore;
+        _Resources[ItemType.Power] = power;
+        OwnerID = -1;
```

Comment:
if we change this to be a reference to the owner rather than an id we should make this null